src/Pages/PDFPreview.js

```diff
@@ -0,0 +1,42 @@
+import { useStage } from '../context/StageContext'
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PDFViewer from '../components/PDFViewer/PDFViewer'
+import { checkout } from '../services/checkoutServices/postToCheckout'
+
+export const PDFPreview = () => {
+
+    const {pdfUrl} = useStage()
+
+	const [redirectUrl, setredirectUrl] = useState(null)
+
+	const goToCheckout = async () => {
+		const url = await checkout('price_1MLxogFPiM3jeCEiCsDS0lDy', pdfUrl)
+		setredirectUrl(url)
+		console.log(url)
+	}
+
+	return (
```

Comment:
We could have the Wrapper as the parent component, then try to use selectors inside the styled component, something like 
const Wrapper = styled.section`
.pdf-container {
 display: grid;
...
}`

Reply:
Also if we can avoid having fragments as parent components would help with readability, less nesting === easier to come back and read

Reply:
Wrapper is now the paren component. Fragment gone!!